module/Kbuild.in

```diff
@@ -41,7 +33,191 @@ ifneq ($(KBUILD_EXTMOD),)
 @CONFIG_QAT_TRUE@KBUILD_EXTRA_SYMBOLS += @QAT_SYMBOLS@
 endif
 
-subdir-asflags-y := $(ZFS_MODULE_CFLAGS) $(ZFS_MODULE_CPPFLAGS)
-subdir-ccflags-y := $(ZFS_MODULE_CFLAGS) $(ZFS_MODULE_CPPFLAGS)
+asflags-y := $(ZFS_MODULE_CFLAGS) $(ZFS_MODULE_CPPFLAGS)
+ccflags-y := $(ZFS_MODULE_CFLAGS) $(ZFS_MODULE_CPPFLAGS)
+
+# Suppress unused-value warnings in sparc64 architecture headers
+ccflags-$(CONFIG_SPARC64) += -Wno-unused-value
+
+
+obj-$(CONFIG_ZFS) := spl.o zfs.o
+
+SPL_OBJS := \
+	spl-atomic.o spl-condvar.o spl-cred.o spl-err.o spl-generic.o \
+	spl-kmem.o spl-kmem-cache.o spl-kstat.o spl-proc.o \
+	spl-procfs-list.o spl-taskq.o spl-thread.o spl-trace.o \
+	spl-tsd.o spl-vmem.o spl-xdr.o spl-zlib.o
```

Comment:
Personally, I'd suggest adopting the style of one object per line as was done in `module/Makefile.bsd`.  Admittedly, that's entirely personally preference so if you feel strongly about it I'm fine with this.

Reply:
I don't, but there's a *lot* and it was easier to navigate; I'll unwind this